site/_posts/2022-06-03-zanocloud-helps-companies-with-fusionauth.md

```diff
@@ -0,0 +1,96 @@
+
+The largest deployment at the current time is for ~9000 users. The projects use either the web-based code grant flow or implement login/token refresh/logout through the login API server-side.
+ 
+**Dan:** What problems did we solve for you?
+
+**Dmitry:** A single auth server for own apps and 3rd party services integrated together (e.g. chats, forums).
+
+Self-hosted deployments are convenient for projects where you want everything to be managed in one environment as well as for dev experiments. And it’s very attractive for early-stage startups when they need to reduce cost of tools and services but are still able to switch to paid plans later if they need more features and support.
+
+**Dan:** How were you solving them before FusionAuth?
+
+**Dmitry:** We used built-in authentication layers of different frameworks (Spring, Dropwizard, Wordpress) or managed auth services.
+
+Built-in authentication layers worked for individual apps but it was hard to integrate them with other applications for single sign-on. Also it didn’t work well for microservices.
+
+Managed auth services handled single sign-on pretty good but it was not always easy to migrate a user base to them. Also, some projects had requirements of local development and deployment without dependencies on external services.
+
+There are some great options on the market that allow companies to overcome the limitations and fit the requirements, but we prefer using FusionAuth for some of our projects.  
+
+> ... it’s very attractive for early-stage startups when they need to reduce cost of tools and services but are still able to switch to paid plans later if they need more features and support.
+
+**Dan:** Why did you choose FusionAuth over the alternatives?
+
+**Dmitry:** We choose FusionAuth because of its installation and configuration simplicity. Self-hosted. Available lambdas and event management.
+
+Seamless user migration from other systems (e.g. we wrote a simple FusionAuth plugin to import WordPress user credentials).
+
+FusionAuth used the same tech stack as many of our projects at the time: Java, Freemarker, and JavaScript.
+
+If there are any questions it’s pretty easy to find information in the FusionAuth documentation. FusionAuth has a great community/forum where you can get support. You can also contact the developers with  feature requests or bug reports and get timely responses.
+
+**Dan:** How much time and money would you say FusionAuth has saved you?
+
+**Dmitry:** It’s hard to estimate, it was different for every project. But I would say, hundreds/thousands of dollars of service cost for the initial project stages compared to managed services.
+
+Time-wise, it saved several weeks of work and provided a better user experience for user base migration to FusionAuth.
+
+> We choose FusionAuth because of its installation and configuration simplicity.
+
+**Dan:** How do you run FusionAuth (kubernetes, standalone tomcat server, behind a proxy, etc)?
```

Comment:
Suggest replace `standalone tomcat server` with `standalone web app`. We are removing Tomcat in a future release, and the type of app server we use is not pertinent.

Reply:
OK